Makefile.inc

```diff
@@ -1,5 +1,6 @@
-# This Makefile is included by src/mainboards/VENDOR/BOARD/Makefile
-TOP         = $(abspath $(CURDIR)/../../../..)
+# This file is included by all Makefiles. The user is required to set
+# the path to OREBOOT.
+TOP         = $(OREBOOT)
```

Comment:
Add something to the top level readme under "Setting up the toolchain" recommending adding export OREBOOT=... to bashrc.

Reply:
Is it possible to automatically determine TOP without the user setting any variables?

Reply:
done. It's automatic now.